src/style.css

```diff
@@ -1,3 +1,70 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
 body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+.navList {
   margin: 0;
```

Comment:
Consider adding `padding: 0;` to the `.navList` class to ensure there is no default padding applied by the browser, which is already done but ensure consistency across all browsers.